db/migrate/20170605090919_add_expression_type_to_miq_alert.rb

```diff
@@ -0,0 +1,23 @@
+class ::MiqExpression; end
```

Comment:
I just remembered where we stubbed classes for deserialization (if that turns out to be needed): https://github.com/ManageIQ/manageiq/blob/9fdcde929728df157bcc1c8fc73fa4f8b29fd60e/spec/migrations/20150625220141_fix_serialized_reports_for_rails_four_spec.rb#L6-L10